step-plans/step-plans-base-artefacts/src/test/java/step/reporting/ResolvedPlanBuilderTest.java

```diff
@@ -306,6 +353,36 @@ public void planWithRecursiveCallPlan() throws IOException, InterruptedException
                         "            Set: 1x: PASSED > nextCount = 5\n" +
                         "            If: 1x: PASSED\n",
                 node.toString());
+
+        // Test partial aggregated tree for single Set
```

Comment:
Not sure to understand what we're exactly testing here. Do we test the case where the node doesn't belong to an "iteration"?